.gitlab-ci.yml

```diff
@@ -228,7 +229,7 @@ run-tests:
     when: always
   tags:
     - targets
-  timeout: 5m
+  timeout: 10m
```

Comment:
If the overall timeout is only 10 minutes, then the timeout of each step should definitely be smaller (see my other comment about the 10 minute timeout).

However, this is a step that we really *don't* want to interrupt, so the timeout here should probably be even larger.

Reply:
Very good point, some of the devices we have take more time than others, and we don't know about the devices we don't support yet.

Ideally this timeout should never be reached, so I'll set it to 20 minutes for now.